v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -102,18 +102,11 @@ final class Offset {
       },
       description = "Kafka Source Authentication Mode",
       helpText =
-          ("The mode of authentication to use with the Kafka cluster. "
-              + "Use "
-              + KafkaAuthenticationMethod.NONE
-              + " for no authentication, "
-              + KafkaAuthenticationMethod.SASL_PLAIN
-              + " for SASL/PLAIN username and password, "
-              + KafkaAuthenticationMethod.TLS
-              + "for certificate-based authentication. "
-              + KafkaAuthenticationMethod.APPLICATION_DEFAULT_CREDENTIALS
-              + " should be used only for Google Cloud Apache Kafka for BigQuery cluster since "
-              + "This allow you to authenticate with Google Cloud Apache Kafka for BigQuery using application default credentials"))
-  @Default.String(KafkaAuthenticationMethod.APPLICATION_DEFAULT_CREDENTIALS)
+          "The mode of authentication to use with the Kafka cluster. "
+              + "Use `NONE` for no authentication, `SASL_PLAIN` for SASL/PLAIN username and password, "
+              + "and `TLS` for certificate-based authentication. "
+              + "Apache Kafka for BigQuery only supports the `SASL_PLAIN` authentication mode.")
+  @Default.String(KafkaAuthenticationMethod.SASL_PLAIN)
```

Comment:
@Dippatel98 Can you verify this change?

Reply:
Reverted the change based on - https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1649